### DISCUSSION TITLE
Input Number Prefix support?

### DISCUSSION THREAD

**Discussion by solmanter** [element-plus/element-plus#13512] [Q&A]
Hi. Is Input number prefix support  or can i put some text inside a input?  
<img width="260" alt="Screenshot 2023-07-07 at 20 34 29" src="https://github.com/element-plus/element-plus/assets/104755033/695981c5-0963-4074-9858-3c319ac7fb85">

**Comment by ryuhangyeong** [comment]
No. Currently not supported.

**Reply by solmanter** [reply]
ok
